### PR TITLE
Update view-reports-for-atp.md

### PR DESCRIPTION
**Pull request by ruiwensu**
URL report - detail view is actually in real time, but aggregate view is not. Also, there's no need to use URL Trace since the details view provides the same capabilities that URL Trace does.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/view-reports-for-atp.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ This report is based data collected, and threats detected, per click (whereas mo
 In the details table, you'll be able to see more information regarding click time and user information. Finally, keep in mind the URL Protection Status report shows the protection from ATP Safe Links feature, so only customers who have enabled ATP Safe Links will see data reflected on this report.
 
 > [!NOTE]
-> This is a *protection trend report*, meaning data represents trends in a larger dataset. Reporting isn't available in real time here. For real time URL click data, please continue to use URL Trace.
+> This is a *protection trend report*, meaning data represents trends in a larger dataset. As a result, the data in the aggregate view is not available in real time here, but the data in the details table view is, so you may see slight discrepancy between the two views.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/view-reports-for-atp.md]:
```suggestion
> This is a *protection trend report*, meaning data represents trends in a larger dataset. As a result, the data in the aggregate view is not available in real time here, but the data in the details table view is, so you may see a slight discrepancy between the two views.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by mypil** [comment]
@ruiwensu  - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@MSFTTracyP - PR has been copy edited and is ready for final review. 


Thank you.

cc: @kenwith

**Pull request merged**